risc0/zkvm/src/lib.rs

```diff
@@ -16,9 +16,9 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![deny(rustdoc::broken_intra_doc_links)]
 #![deny(missing_docs)]
+#![feature(array_try_from_fn)]
```

Comment:
This is a feature that's unstable that I used to return an error status when building an array. Not sure if this is ok but I thought I'd point it out.

Reply:
So far the policy has been to not use any unstable features unless there's no other option. Since we're using a nightly version of the rust compiler, technically this would work. However, I'm hoping we can move to a stable compiler eventually. At least on the host-side, it should be totally fine to use a stable compiler, it's just a hassle to setup a separate toolchain for host vs guest.

Reply:
Ok, I'll look at other options!